src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/js/search.js

```diff
@@ -115,7 +116,8 @@ require(["fuse"], function (Fuse) {
     RESULTS.style.display = "flex";
     const warningBanner = document.createElement("div");
     warningBanner.className = "warning-banner";
-    warningBanner.textContent = "No results found.";
+    warningBanner.textContent =
+      "No results found. Press Enter for page search.";
```

Comment:
Maybe something like:

```suggestion
    warningBanner.textContent =
      "No results found. Press Enter for extended search.";
```